extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventCommonFieldsMixin.schema.json

```diff
@@ -383,6 +383,16 @@
               "type": "string",
               "meta:titleId": "journeyStepEventCommonFields##https://ns.adobe.com/experience/journeyOrchestration/eventName##title##18181",
               "meta:descriptionId": "journeyStepEventCommonFields##https://ns.adobe.com/experience/journeyOrchestration/eventName##description##91181"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/origTimestamp": {
+              "title": "Projection Original Timestamp",
+              "type": "string",
+              "description": "The Projection Original Timestamp of the segment being used in journey"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/_id": {
```

Comment:
This is extending Exp event class which comes with _id too . How is this _id different from EE _id ?

Reply:
The current EE _id  is the event id. 
The new _id we are proposing within experience is the experience id that comes from Projections.

Reply:
got it , any reason why you wanted it to be _id instead of more descriptive name ?

Reply:
We were thinking about experienceID, but this is already within experience tree so thought to make it _id

Let me know if you want to change it to more descriptive name

Reply:
experienceId looks good. as a convention in our events, we don't suffix with `ID`, but `Id`. In step events, it will be `experienceID`, and in our event, it should be `experienceId`

Reply:
experienceID looks good for me